src/main/kotlin/org/rust/ide/formatter/settings/RustLanguageCodeStyleSettingsProvider.kt

```diff
@@ -32,18 +32,27 @@ class RustLanguageCodeStyleSettingsProvider : LanguageCodeStyleSettingsProvider(
                     "KEEP_BLANK_LINES_IN_DECLARATIONS",
                     "KEEP_BLANK_LINES_IN_CODE")
 
-            WRAPPING_AND_BRACES_SETTINGS ->
+            WRAPPING_AND_BRACES_SETTINGS -> {
                 consumer.showStandardOptions(
                     "RIGHT_MARGIN",
-                    "ALIGN_MULTILINE_CHAINED_METHODS");
+                    "ALIGN_MULTILINE_CHAINED_METHODS",
+                    "ALIGN_MULTILINE_PARAMETERS",
+                    "ALIGN_MULTILINE_PARAMETERS_IN_CALLS")
+
+                consumer.showCustomOption(RustCodeStyleSettings::class.java,
+                    "ALIGN_RET_TYPE_AND_WHERE_CLAUSE",
+                    "Align return type and where clause",
+                    CodeStyleSettingsCustomizable.WRAPPING_METHOD_PARAMETERS)
+            }
         }
     }
 
     override fun getIndentOptionsEditor(): IndentOptionsEditor? = SmartIndentOptionsEditor()
 
     override fun getDefaultCommonSettings(): CommonCodeStyleSettings =
         CommonCodeStyleSettings(language).apply {
-            RIGHT_MARGIN = 99
+            RIGHT_MARGIN = 100
```

Comment:
Yep, looks like it. [Styleguide] says 99, but I think it is wrong. It makes sense to use 79 instead of 80 to save one column for the terminal text editor info, but this argument does not work for 99 vs 100

[Styleguide]: https://aturon.github.io/style/whitespace.html